src/bridge/cockpitrouter.c

```diff
@@ -515,6 +526,21 @@ cockpit_router_init (CockpitRouter *self)
 }
 
 static void
+cockpit_router_ban_hosts (CockpitRouter *self)
+{
+  RouterRule *rule;
+  JsonObject *match = json_object_new ();
```

Comment:
Initially that's what I had. But cockpit-stub initially didn't load bridges. So hosts were falling through to the regular channel processing, leading to weird hard to debug results. Since hosts is so fundamental to our protocol, i figured that we should just refuse by default the way we used to even if no bridges get loaded.